src/main/scala/nl.knaw.dans.easy.dd2d/DatasetEditor.scala

```diff
@@ -133,11 +133,12 @@ abstract class DatasetEditor(instance: DataverseInstance, optFileExclusionPatter
     } yield ()
   }
 
-  protected def getAllFiles(persistendId: PersistendId): Try[List[FileMeta]] = {
+  protected def getFilesToEmbargo(persistendId: PersistendId): Try[List[FileMeta]] = {
     for {
       r <- instance.dataset(persistendId).listFiles()
       files <- r.data
-    } yield  files
+      filesToEmbargo = files.filter(f => f.directoryLabel.getOrElse(true).toString != "easy-migration")
```

Comment:
If there is no directoryLabel you will end up comparing "true" != "easy-migration" which doesn't make any sense.

I think you need something like:

f.directoryLabel.map(_ != "easy-migration").getOrElse(true)

Reply:
Autch, it had to of course be  a string....